Edge10.CouchDb.Client/KeyParameter.cs

```diff
@@ -11,12 +11,15 @@ public class KeyParameter
 
 		public static implicit operator KeyParameter(string key)
 		{
+			if (key == null)
```

Comment:
Yeh I misread the following method `KeyParameter` as `ToKeyParameter` :man_facepalming: